### PR TITLE
Create 2_new_data_model.md

### PR DESCRIPTION
**Pull request by siatwangmin**
fix some typo

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

there are some typo at "mask\_rcnn\_r50\_caffe\_fpn\_mstrain-poly\_1x\_balloon.py"  **dataset\_type** **\=** 'COCODataset'

## Modification

there are some typo at "mask\_rcnn\_r50\_caffe\_fpn\_mstrain-poly\_1x\_balloon.py"  **dataset\_type** **\=** 'COCODataset'

## Checklist

1.  Pre-commit or other linting tools are used to fix the potential lint issues.
2.  The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3.  If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4.  The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by hhaAndroid — APPROVED** [review]

File: docs_zh-CN/2_new_data_model.md [hunk 1]
```diff
@@ -58,6 +59,7 @@ categories = [{
     "supercategory": str,
 }]
 ```
+
 现在假设我们使用 balloon dataset。
 
 下载了数据集之后，我们需要实现一个函数将标注格式转化为 COCO 格式。然后我们就可以使用已经实现的 `COCODataset` 类来加载数据并进行训练以及评测。
```
Comment (RangiLyu) [inline docs_zh-CN/2_new_data_model.md:65]:
```suggestion
下载了数据集之后，我们需要实现一个函数将标注格式转化为 COCO 格式。然后我们就可以使用已经实现的 `CocoDataset` 类来加载数据并进行训练以及评测。
```

**Comment by ZwwWayne** [comment]
Kindly ping @siatwangmin , we can merge this PR after resolving comments.

**Review by ZwwWayne — APPROVED** [review]

**Comment by ZwwWayne** [comment]
Can be merged after resolving conflicts.

**Comment by OpenMMLab-Coodinator** [comment]
Hi siatwangmin！First of all, we want to express our gratitude for your significant PR in the MMDet project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! **Join us ：https://discord.gg/raweFPmdzG**

If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**